website/content/docs/internals/integrated-storage.mdx

```diff
@@ -130,8 +140,54 @@ will be encrypted by Vault's barrier.
 
 Vault does not currently offer automated dead server cleanup. If you wish to
 decommission a node, or a node dies and must be replaced, the node must manually
-be removed from the cluster with the `remove peer`
-[command](/docs/commands/operator/raft#remove-peer).
+be removed from the cluster with the `remove peer` [command](/docs/commands/operator/raft#remove-peer).
+
+### Quorum Management
+
+#### Autopilot
+
+An [Autopilot feature](https://www.vaultproject.io/docs/concepts/integrated-storage/autopilot) 
+is available since 1.7.x & later versions that include configurable parameters 
+for when a node is treated as healthy before it's considered an eligible voter in the 
+Quorum list. Other features which may be enabled include the ability to remove nodes 
+considered as dead from the Quorum list after a certain period.
+
+Autopilot is enabled by default in Vault 1.7+  with default 
+values that should be adapted to best suit the environmental requirements where 
+Vault instances are deployed.
+
+Autopilot includes stabilization logic for nodes joining the cluster.
+Recently joined nodes are 
+accepted as non-voter initially until they are in sync with matching Raft index 
+and only after a stability thresholds are they then full voting members.
+Setting the stability threshold too low can result in cluster instability as nodes will be
+counted as voters before they are capable of voting.
+
+#### Without Autopilot
+
+Older versions of Vault, 1.6.x & lower, as well as cases where Autopilot may be 
+disabled or misconfigured, behave differently.
+
+In scenarios involving those when a node joins a Raft cluster, it attempts to 
+catch up with the reset of the nodes through the data that it's replicating from 
+the leader. While in this initial synchronisation state, the node cannot 
+vote but is counted for the purposes of quorum. If a number of new nodes join 
+the cluster simultaneously or at similar times, and thereby exceeding the failure
+tolerance of the cluster, quorum may be lost and the cluster can fail.
+
+For example, consider a scenario where there is a 3-node cluster with a large 
+amount of data and a failure tolerance of 1. An additional 3 new nodes then 
+join the cluster. The cluster now consists of 6 nodes with a failure tolerance 
+of 2, but since all 3 nodes are still catching up, this will result in a loss of 
+quorum.
+
+* A 3 node cluster with a large amount of data that's at a failure tolerance of 1.
+* Another 3 new nodes then join the cluster together.
+* Now the cluster consists of 6 nodes with a failure tolerance of 2, but all 3 new nodes are still catching up, resulting in a loss of quorum.
```

Comment:
These bullet points feel a little redundant, as they say the same thing as the preceding paragraph.

Reply:
A little repeat is not bad - some people dont read sentence as well as bullets and these are summaries of the same thing. I dont see any issue as it re-enforces the earlier sentence more briefly - lets keep it.